.golangci.yml

```diff
@@ -38,6 +41,7 @@ linters:
     # - whitespace
     # - wsl
     # - gocognit
+    # - nolintlint
```

Comment:
the reviewdog docker image of golangci-lint hasnt been updated to the latest version so commenting it out for now.